### PR TITLE
feat: added fetching from /rpc to get actual data on the application's flags

### PR DESCRIPTION
**Pull request by Xenofic**
by using :id/rpc from discord which is a undocumented endpoint we can get the application data which also includes it's intents, so in general we can have actual display of the client's information

### CODE REVIEW

**Comment by Dqrshan** [comment]
Since you're fetching privileged intents from `/rpc` and displaying them as enabled/disabled, in the existing code we're displaying intents that the bot currently has in its [`ClientOptions`](https://old.discordjs.dev/#/docs/discord.js/main/typedef/ClientOptions). 

Let's try a different approach where we merge both and display the privileged intents **AND** the client's intents that are related to the privileged intents. 

For example: I have `MessageContent` privileged intent enabled, but I've not passed it into my **ClientOptions**. In such a case, it would be informative to display such information.

File: src/commands/info.ts [hunk 1]
```diff
@@ -68,20 +67,50 @@ const info = async (
...
+                    (k) =>
+                        `- ${
+                            flags[
+                                k as
+                                    | 'GuildMembers'
+                                    | 'GuildPresences' as 'MessageContent'
+                            ]
+                                ? '✅'
+                                : '❌'
+                        } \`${k}\``
```
Comment (Dqrshan) [inline src/commands/info.ts:114]: review based on [this](https://github.com/Dqrshan/discord-debug/pull/6#issuecomment-1646582859)

**Comment by Xenofic** [comment]
Let's have a configurable approach for it then, the users can set it accordingly?

**Comment by Dqrshan** [comment]
Why configurable? It makes sense to show both the privileged intents and the client intents.

**Pull request closed without merging**